doc/lua_api.txt

```diff
@@ -2220,6 +2220,16 @@ Elements
 * `frame duration`: Milliseconds between each frame. `0` means the frames don't advance.
 * `frame start` (Optional): The index of the frame to start on. Default `1`.
 
+### `mesh[<X>,<Y>;<W>,<H>;<mesh name>;<texture names>;<rotation X>,<rotation Y>;<continuous rotation>;<mouse control>]`
+
+* Show a mesh model.
+* `mesh name`: The mesh model to use.
+* `texture names`: The mesh textures to use according to the mesh materials.
+   Texture names must be separated by commas.
+* `rotation {X,Y}` (Optional): Set a given rotation to the camera.
```

Comment:
This "rotation" is euler angles in degrees (http://irrlicht.sourceforge.net/docu/classirr_1_1core_1_1_c_matrix4.html#a8ee5ef8619d4b0f56d72ac84495ed644), right?
It might be worth mentioning this, or that it's the same sort of rotation as in object `set_rotation` (only if it really is the same).